src/coreclr/gc/gc.cpp

```diff
@@ -6217,7 +6339,12 @@ class heap_select
         if (GCToOSInterface::CanGetCurrentProcessorNumber())
         {
             uint32_t proc_no = GCToOSInterface::GetCurrentProcessorNumber();
-            return proc_no_to_heap_no[proc_no];
+            int best_heap = proc_no_to_heap_no[proc_no];
```

Comment:
I wouldn't call this `best_heap` which makes it sound like this is just a perf thing. but it wouldn't work if we didn't do `%`. probably call it something like `adjusted_heap`.

Reply:
ok.